clevertap-core/src/main/java/com/clevertap/android/sdk/pushnotification/fcm/FcmSdkHandlerImpl.java

```diff
@@ -73,6 +73,12 @@ public boolean isSupported() {
 
     @Override
     public void requestToken() {
+        if (!Utils.haveDeprecatedFirebaseInstanceId){
+            config.getLogger().debug(config.getAccountId(),"It looks like you're using the " +
+                    "latest FCM dependency.Downgrade you're FCM dependency to v20.2.4");
```

Comment:
@william-ct Make this "Downgrade you're FCM dependency to v20.2.4 or else CleverTap SDK will not be able to generate a token for this device."